pkg/kv/migration.go

```diff
@@ -23,7 +23,6 @@ func NewDatabaseMigrator(params kvparams.KV) *DatabaseMigrator {
 	}
 }
 
-// Migrate TODO (niro): Currently just set up the KV DB Schema version. Need to create a migration flow for KV
```

Comment:
This comment is still relevant. We are not really doing a migrate process here but simply writing the initial KV version. We will need to modify that